pup-0006.md

```diff
@@ -0,0 +1,100 @@
+describe giving the commit bit to a contributor. The current committers want to openly document
+the process.
+
+
+## Motivation
+
+Comitter - is an individual who has write permissions to a
+[Pulp Organization](https://github.com/pulp/) repository.
+To have "a commit bit" means one is permitted to commit those changes.
+
+Historically the commit bit was given on day 0 when hired on the core team at Red Hat.  In Oct 2017
+the existing committers decided to stop that practice and instead document an open process.
+Engineers hired on the Pulp team since Oct '17 have not received commit bit.
+
+We have not yet documented an open process of which to become a committer.
+The process outlined in this document is inspired by the processes used in Chromium projects and
+Apache Software Foundation.
+
+
+## Scope
+
+This policy is intended as the default for all software in the
+[Pulp Organization](https://github.com/pulp/) unless explicitly stated otherwise.
+
+
+## Becoming a committer.
+
+Overall the candidate must have demonstrated:
+* project commitment, active involvement and care towards the needs of all Pulp users and not only
+  their own interests
+* contribution of constructive code reviews
+* the ability to collaborate with the team and understand its policies and workflows
+* understanding of the projects' code base and functionality by submitting code that was accepted
+  and merged in the repositories of interest
+
+These requirements are somewhat vague by design and leave the decision to the judgement of the
+existing committers. Users rely on the judgement of committers already, so to rely on that judgement
+to accept new contributors is reasonable. Anyone who specifically wants to get more involved should
+approach the existing commit bit holders about mentorship.
+
+
+## Process
+
+A nomination or self-nomination email should be sent to the pulp-dev email list and must contain
+the following:
+
+* Name and email of the nominee. This can be a self-nomination.
+* Explanation of why the nominee should be a committer.
+* Repository being requested.
+* Vote end date. Must be 7 calendar days from now.
+
+To pass, a +1 vote is required from the majority of existing committers for that repository and
+no -1 votes.
+
+If a nomination does not pass, the candidate may be re-nominated after a cool down period of 60
+days.
+
+
+# Revocation of the committer status
+
+If there are problems or disagreements within the community, they can usually be solved through
+open discussion and debate.
+
+In the unhappy event that a committer continues to disregard good citizenship (or actively disrupts
+the project), that person's status may be revoked. The process is similar as for nominating a new
+committer with a +1 voting of two-thirds of the existing committers for that repository.
+
+In case of voluntary resignation of the commit bit, an email should be sent with the notification and
+reasoning. No futher voting should take place.
```

Comment:
I might would add some portion of lack of activity.  (for example:. a committer has not had some activity after 6 months)

Reply:
I'm sentimental towards the idea that committers should be active, but in terms of requirements I think we should rely on the normal removal process described "Revocation of the committer status" to handle cases where committers go dark for too long. A requirement of activity could easily be satisfied with a single comment or PR review.

Reply:
In my understanding it is implied that in order to get eligible to get the commit bit it is required to be active, then to maintain the status it's also common sense to continue to be active as well as continuing to demonstrate [0]
[0] https://github.com/pulp/pups/pull/15/files#diff-524cae13395c2799735cd173283a43faR39